docs/instruments/ZJSONInstrument_v0.0.1.md

```diff
@@ -0,0 +1,1028 @@
+            "longName": {
+                ...
+            },
+            "multilingual": boolean,
+            "previousVersion": string,
+            "schemaVersion": string,
+            "shortName": string
+        },
+        ...
+    },
+    ...
+}
+```
+
+The metadata object contains useful information such as the version of the instrument, its name, its description, and
+some provenance data. The value of each key is defined as such:
+
+`meta.createdBy`: String.
+
+The developer or user responsible for creating the Z-JSON Instrument e.g. the Z-JSON Instrument of a paper instrument,
+created by some study, will be `createdBy` *this* LORIS developer.
+
+`meta.createdOn`: String.
+
+The date of creation of the Z-JSON Instrument. Each new version of an instrument will have a new `createdOn` date. Small
+changes, such as bug fixes, that do not require the creation of a new verion of an instrument should use `meta.lastUpdateOn`
+instead to track updates.
+                 
+`meta.createdWith`: String.
+
+The software or tool responsible for the creation of the ZJSON Instrument i.e. the Instrument Builder module or some conversion script.
+
+`meta.defaultLanguage`: Required String.
+                        
+The default (human communication) language of the instrument, represented by an IETF BCP 47 (RFC 5646 Standard) language tag e.g. "en-CA".
+It is important to set the default language as it is used as a key in other locations of the `schema` object. This key is required to be compatible with
+ReproNim's JSON-LD Instrument.
+
+`meta.derivedFrom`: String.
+                    
+The resource instrument that the Z-JSON Instrument derived from as a result of significant content modification such that the instrument is no longer the same                    
+instrument, and the creation of a new version does not suffice.
+                    
+This level of modification does not include format transcription i.e. linst to
+JSON, unless the content of the instrument has also been modified. The
+`meta.importedFrom` key should be used in the case of conversions.
+
+`meta.importedFrom`: String.
+
+The resource instrument that this Z-JSON Instrument was converted from, given
+that the imported content is preserved, and only its format is modified.
+
+If the Z-JSON instrument is an original instrument, the value of this key is
+null.
+
+`meta.includeMetaDataFields`: Boolean. 
+
+True of false that the Z-JSON Instrument will include LORIS-specific
+metadata fields, such as 'Candidate Age', or 'Examiner', which
+calculates the candidate's age, and selects the examiner
+administering the instrument, respectively.  
+
+`meta.instrumentVersion`: Required string.
+
+The version of the instrument being described.
+
+It follows some convention as determined by the author of the instrument,
+starting with some representation of the number 1 and incrementing itself alongside any changes made to the `schema` object. It is self-contained and
```

Comment:
do versions not have to increment? i put down 'representation' of the number 1, and not the number itself. so e.g. v0.0.1 would be a representation of 1. I thought the point of versioning is that you can see how far from 1 that version is and see the lineage through numbers

Reply:
i think an implementation would need to set a standard for their version string. otherwise, could we set a standard? not sure if this would be compatible with existing versions of current instruments